app/design/adminhtml/default/default/template/page/footer.phtml

```diff
@@ -29,12 +29,9 @@
     <?php echo $this->__('Interface Locale: %s', $this->getLanguageSelect()) ?>
 </p>
 <p class="legality">
-    <a href="<?php echo $this->getConnectWithMagentoUrl() ?>" id="footer_connect"><?php echo $this->__('Connect with the Magento Community') ?></a><br/>
-    <img src="<?php echo $this->getSkinUrl('images/varien_logo.gif') ?>" class="v-middle" alt="" />&nbsp;&nbsp;
-    <?php echo $this->__('Magento&trade; is a trademark of Magento Inc.<br/>Copyright &copy; %s Magento Inc.', date('Y')) ?>
+    <a href="<?php echo $this->getConnectWithMagentoUrl() ?>" id="footer_connect"><?php echo $this->__('Connect with the OpenMage Community') ?></a>
 </p>
-<?php echo $this->__('OpenMage ver. %s', Mage::getOpenMageVersion()) ?>
-&nbsp;(<?php echo $this->__('based on Magento ver. %s', Mage::getVersion()) ?>)
```

Comment:
The "based on" part was to avoid the big confusion, which magento version this is based on.
We should keep this for minimum till M1 is eol, better would be to still keep it for a year